containerapps/README.md

```diff
@@ -0,0 +1,22 @@
+# Deploying on Azure Container Apps
+
+Due to [a limitation](https://github.com/Azure/azure-dev/issues/2736) of azd, the azure.yaml file for deploying to Azure Container Apps lives here.
```

Comment:
To reduce redundant documentation, can you move any relevant info into docs/azure_container_apps.md, and just put a pointer here "Please see the guide for deploying to Azure Container Apps"?
I want there to be a single place for the information, so developers know where to go, and we know what to update.

Reply:
Done.